src/main/java/model/GetAllSectionsRequest.java

```diff
@@ -0,0 +1,25 @@
+package model;
```

Comment:
I don't think we need this class, since there isn't a specific input we need to query the sections. We're sending back all the sections in the sections table.